src/sentry/notifications/notifications/daily_summary.py

```diff
@@ -64,6 +64,6 @@ def build_attachment_title(self, recipient: RpcActor) -> str:
         return ""
 
     def build_notification_footer(self, recipient: RpcActor, provider: ExternalProviders) -> str:
-        return (
-            "Getting this at a funky time? This sends at 4pm for whatever time zone you have set."
-        )
+        url_str = "/settings/account/notifications/"
```

Comment:
Instead of hardcoding this, could you use the url thing like for testing with `sentry-account-settings-notifications`? I'm not exactly sure how it works, but I think it'd be safer in the long run

Reply:
ok this was a little strange, we don't have a path for `settings/account/details`, there must be a redirect somewhere because we only have `settings/account` [here](https://github.com/getsentry/sentry/blob/master/src/sentry/web/urls.py#L461-L465). I confirmed going to https://sentry.sentry.io/settings/account/ redirects to https://sentry.sentry.io/settings/account/details

Reply:
sorry I'm a bit confused, the string was originally `/settings/account/notifications/` so does the path for `/settings/account/details` work?

Reply:
because I was wrong before 😭 the notifications path is for like, alert rule settings and stuff, this is the one I actually wanted for the top level user settings that includes your timezone